feature_layers/display-subtype-feature-layer/src/main/resources/style.css

```diff
@@ -0,0 +1,27 @@
+.panel-region .label {
+  -fx-text-fill: white;
+}
+
+.label {
+  -fx-text-fill: black;
+}
+
+.slider .axis {
+    -fx-tick-label-fill: white;
+}
+
+.range-slider .axis {
+    -fx-tick-label-fill: white;
+}
+
+.panel-region .check-box {
```

Comment:
I think you're only using the `.panel-region *` rules in this style.css, so would be worth getting rid of the rest. I think the idea was to keep the style.css files limited to only the used rules going forward, and some day go back and tidy up the old ones.